helm/hpcc/values.schema.json

```diff
@@ -720,6 +720,11 @@
         "logging": {
           "$ref": "#/definitions/logging"
         },
+        "preload": {
+          "description": "Preloaded plugins",
+          "type": "array",
```

Comment:
This probably should be documented somewhere - e.g., that the lib and .so are not needed.